README.md

```diff
@@ -95,3 +94,13 @@ Once you have successfully provisioned a GitPod environment (see [Getting Starte
 - [Part 2: Using Bash scripting](docs/part2-bash.md)
 - [Part 3: Using Nextflow](docs/part3-nextflow.md)
 - [Part 4: Using the Seqera Platform](docs/part4-platform.md)
+
+## Resources
```

Comment:
Wondering it is worth just linking out to a manual of other basic UNIX commands like this: https://mally.stanford.edu/~sr/computing/basic-unix.html